.travis.yml

```diff
@@ -37,6 +41,7 @@ jobs:
           - determineScalaVersion
           - deriveModuleVersions
           - removeExistingBuilds $integrationRepoUrl
+          - clearIvyCache
```

Comment:
Where is ivydir set on travis?

Reply:
`common` sets `IVY2_DIR="$WORKSPACE/.ivy2"`, where `WORKSPACE="${WORKSPACE-`pwd`}"`. I'll add that to the env var comment above.